zk/hermez_db/db.go

```diff
@@ -508,29 +486,6 @@ func (db *HermezDb) DeleteL1BlockHashGers(l1BlockHashes *[]common.Hash) error {
 	return nil
 }
 
-func (db *HermezDb) WriteL1BlockHash(l1BlockHash common.Hash) error {
-	return db.tx.Put(L1_BLOCK_HASHES, l1BlockHash.Bytes(), []byte{1})
-}
-
-func (db *HermezDbReader) CheckL1BlockHashWritten(l1BlockHash common.Hash) (bool, error) {
```

Comment:
If we don't need these tables anymore, might as well remove from tables.go in cdk-erigon-lib.

Reply:
Yep we can likely remove these.  For now the tables are still created by the call to handle the hermezdb though.  So we can tackle that later